featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -588,17 +589,17 @@ def _get_ctes_with_casted_data(
         # get subquery with columns casted to string to compute value counts
         casted_columns = []
         for col_expr in sql_tree.expressions:
-            col_name = col_expr.alias or col_expr.name
+            alias_or_name = col_expr.alias or col_expr.name
```

Comment:
sorry i meant whether `col_expr` has the property `alias_or_name` here, so that ou can do `col_expr.alias_or_name` instead

Reply:
Ah I see, yes I think that should work. Will update.